Source/Core/VideoBackends/D3D/Render.cpp

```diff
@@ -162,6 +159,21 @@ static void SetupDeviceObjects()
   D3D::SetDebugObjectName(s_reset_rast_state, "rasterizer state for Renderer::ResetAPIState");
 
   s_screenshot_texture = nullptr;
+
+  D3D11_BLEND_DESC blenddesc_xfb;
```

Comment:
Use the VideoCommon union here to avoid duplicating the blend state config.

Reply:
Good catch, could have sworn I was already using the videocommon struct..